sceptre/cli/list.py

```diff
@@ -127,3 +126,32 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.command(name="stacks")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_stacks(ctx, path):
+    """
+    List stack names and command paths for stacks in a
+    stack group.
```

Comment:
The docs for this command is a bit unclear because stack names can be retrieved from local or remote.  It can also be derived by calculating a union of local and remote stacks.  This leads me to wonder about the purpose and intention of this command.  It looks like it currently only shows a list of locally defined stacks in a stack group but is that the only thing it should do?  if so then could you please clarify that in the docs?  

On a side note, it would be nice to exclude from the list the local stacks that have not been deployed to AWS.  Although this can be done in a separate PR.